workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/WorkerInspectorWindow.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Editor.UIElements;
+using Unity.Entities;
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector
+{
+    internal class WorkerInspectorWindow : EditorWindow
+    {
+        private WorldSelector worldSelector;
+        private EntityList entityList;
+
+        [MenuItem("SpatialOS/Window/Worker Inspector", false)]
+        public static void ShowWindow()
+        {
+            var windowType = typeof(EditorWindow).Assembly.GetType("UnityEditor.InspetorWindow");
```

Comment:
`InspetorWindow`? Also I vaguely remember that this hack was no longer required?

Reply:
Lmao that explains why it was popping up into its own window

Reply:
Also why was this a hack? Its just to say 'dock next to the inspector'